src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6870,28 +6870,43 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
     <value>At least one top-level statement must be non-empty.</value>
   </data>
   <data name="ERR_LineDoesNotStartWithSameWhitespace" xml:space="preserve">
-    <value>Line does not start with the same whitespace as the last line of the raw string literal</value>
+    <value>Line does not start with the same whitespace as the last line of the raw string literal.</value>
   </data>
   <data name="ERR_RawStringNotInDirectives" xml:space="preserve">
-    <value>Raw string literals are not allowed in preprocessor directives</value>
+    <value>Raw string literals are not allowed in preprocessor directives.</value>
   </data>
   <data name="ERR_RawStringDelimiterOnOwnLine" xml:space="preserve">
-    <value>Raw string literal delimiter must be on its own line</value>
+    <value>Raw string literal delimiter must be on its own line.</value>
   </data>
   <data name="ERR_TooManyQuotesForRawString" xml:space="preserve">
-    <value>Too many closing quotes for raw string literal</value>
+    <value>The raw string literal does not start with enough quote characters to allow this many consecutive quote characters as content.</value>
+  </data>
+  <data name="ERR_TooManyOpenBracesForRawString" xml:space="preserve">
+    <value>The interpolated raw string literal does not start with enough '$' characters to allow this many consecutive opening braces as content.</value>
+  </data>
+  <data name="ERR_TooManyCloseBracesForRawString" xml:space="preserve">
+    <value>The interpolated raw string literal does not start with enough '$' characters to allow this many consecutive closing braces as content.</value>
+  </data>
+  <data name="ERR_NotEnoughQuotesForRawString" xml:space="preserve">
+    <value>Not enough quotes for raw string literal.</value>
+  </data>
+  <data name="ERR_NotEnoughCloseBracesForRawString" xml:space="preserve">
+    <value>The interpolation must end with the same number of closing braces as the number of '$' characters that the raw string literal started with.</value>
+  </data>
+  <data name="ERR_CannotMixVerbatimAndRawStrings" xml:space="preserve">
+    <value>Raw string literals cannot be verbatim ('@'). If needed, increase the number of quote characters starting and ending the literal.</value>
   </data>
   <data name="ERR_UnterminatedRawString" xml:space="preserve">
-    <value>Unterminated raw string literal</value>
+    <value>Unterminated raw string literal.</value>
```

Comment:
Periods belong on sentences, but this is a noun phrase, not a sentence. It could be a sentence, for consistency with the surrounding messages, but then it needs a verb. Maybe:

`The raw string literal is not terminated.`